dealership/contracts.py

```diff
@@ -1,12 +1,48 @@
 class Contract(object):
-    pass
-
+    def __init__(self, customer, vehicle):
+        self.customer = customer
+        self.vehicle = vehicle
+        self.interest = {
+            'Car': 1.07,
+            'Motorcycle': 1.03,
+            'Truck': 1.11
+        }
+        if self.customer.is_employee():
+            self.discount = 0.10
+        else:
+            self.discount = 0.0
 
 class BuyContract(Contract):
     def __init__(self, vehicle, customer, monthly_payments):
-        pass
+        Contract.__init__(self, customer, vehicle)
```

Comment:
You should instead be using super to access the parent class:
`super(BuyContract, self).__init__(customer, vehicle)`